File: api/TWTodos/tables/Resposta.cs
public class Resposta
{
    public int IdResposta { get; set; }
    public int IdPostagem { get; set; }
    public Postagem Postagem { get; set; }
    public int IdUsuario { get; set; }
    public Usuario Usuario { get; set; }
    public string Conteudo { get; set; }
    public DateTime DataComentario { get; set; }
}

File: CotuTalk_Program/app/src/main/java/com/example/cotutalk_program/AcessoAPI/data/Resposta.kt
package com.example.cotutalk_program.AcessoAPI.data
import com.google.gson.annotations.SerializedName
import org.threeten.bp.LocalDateTime

data class Resposta(
    val IdResposta: Int,
    val IdPostagem: Int,
    val IdUsuario: Int,
    @SerializedName("conteudo")
    val Conteudo: String,
    val DataComentario: LocalDateTime?
)
